src/api/component.js

```diff
@@ -127,10 +105,11 @@ export default class extends HTMLElement {
   static get observedAttributes () {
     const attrsOnCtor = this.hasOwnProperty($ctorObservedAttributes) ? this[$ctorObservedAttributes] : [];
 
-    const props = getPropsMap(this);
-    const attrsFromLinkedProps = Object.keys(props).map(key => {
-      const { attribute } = props[key];
-      return attribute === true ? dashCase(key) : attribute;
+    const propDefs = getPropsMap(this);
+    // todo: with Object.keys we skip symbol properties!
```

Comment:
Well since we say that symbol property cannot have linked attribute then, it is correct to only loop on properties that are not symbols.